minutes/board_2019_01_09.md

```diff
@@ -0,0 +1,89 @@
+# NiT Board Meeting 01/09/2020
+
+### Agenda
+
+### Attendance
+
+* Adam Recvlohe
+* Nick Sahler
+* Coty Sutherland
+
+### Items from Previous Meeting
+
+* Adam
+    * Bank account update (Oklahoma Central Credit Union)
+        * Received information back from Oklahoma Central Credit Union
+        * They need id’s of board members
```

Comment:
There's a couple of places that are spelled "id's" but it should be "ids" or "IDs".